lib/src/revset.rs

```diff
@@ -2053,12 +2164,14 @@ impl<'a> RevsetParseContext<'a> {
         user_email: String,
         extensions: &'a RevsetExtensions,
         workspace: Option<RevsetWorkspaceContext<'a>>,
+        mailmap: Rc<Mailmap>,
```

Comment:
nit: `Rc<_>` isn't needed, but templater depends on it. If `Mailmap` is cached by `WorkspaceCommandHelper`, we can replace `Rc` with `&'a`. That said, this isn't important, and can be addressed later. (I just find it looked a bit odd because the revset module doesn't require `Rc`.)